source/elements/oneDAL/source/programming_model/computational_modes.rst

```diff
@@ -6,10 +6,22 @@ Computational modes
 Batch
 -----
 
+In the batch processing mode, the algorithm works with the entire data set to produce the final
```

Comment:
not all my comments addressed, I want 
- a single style of 'defs' that would be applied in the entire doc
- short intro to the section
- overall view of the docs as I still do not see why one the most important features of the library is not highlighted immediately in the intro section, but is moved somewhere to the next level of details

Reply:
For last comment  - just to be clear, what is most important features of library? different computational modes?

Reply:
fixed based on offline feedback